package.json

```diff
@@ -0,0 +1,163 @@
+{
+  "name": "wiredcraft-test",
+  "version": "0.0.1",
+  "description": "",
+  "main": "server.js",
+  "scripts": {
+    "test": "jest --verbose",
+    "watch": "jest --verbose --coverage --watchAll",
+    "build": "cross-env NODE_ENV=production webpack --progress --mode=production",
+    "dev": "cross-env NODE_ENV=development webpack --mode=development",
+    "startDev": "npm run dev && webpack-dev-server --content-base dev/ --progress --useLocalIp --host 0.0.0.0 --mode=development --open",
+    "dev-mock": "cross-env MOCK=true webpack-dev-server --content-base dev/ --progress --useLocalIp --host 0.0.0.0 --mode=development --open",
+    "standard": "standard --verbose | snazzy || exit 0",
+    "start": "node server.js"
+  },
+  "keywords": [
+    "wiredcraft",
+    "test",
+    "react"
+  ],
+  "repository": {
+    "type": "git",
+    "url": ""
+  },
+  "author": "xiaoT",
+  "license": "ISC",
+  "dependencies": {
+    "@reduxjs/toolkit": "^1.4.0",
+    "antd": "^4.2.0",
+    "area-data": "^5.0.6",
+    "axios": "^0.18.1",
+    "history": "^4.7.2",
+    "http": "0.0.1-security",
+    "koa": "^2.11.0",
+    "koa-router": "^7.4.0",
+    "koa-static": "^5.0.0",
+    "lodash": "^4.17.19",
+    "normalize.css": "^8.0.0",
+    "prop-types": "^15.6.2",
+    "react": "^16.13.1",
+    "react-area-linkage": "^3.0.0",
+    "react-dom": "^16.13.1",
+    "react-redux": "^7.2.0",
+    "react-router": "^5.0.1",
+    "react-router-dom": "^5.0.1",
+    "typescript": "^3.8.3"
+  },
+  "devDependencies": {
+    "@babel/core": "^7.0.0",
+    "@babel/plugin-proposal-class-properties": "^7.0.0",
+    "@babel/plugin-syntax-dynamic-import": "^7.0.0",
+    "@babel/preset-env": "^7.0.0",
+    "@babel/preset-react": "^7.0.0",
+    "@testing-library/react": "^10.4.7",
+    "@types/enzyme": "^3.10.5",
+    "@types/jest": "^25.2.1",
+    "@types/react": "^16.9.34",
+    "@types/react-dom": "^16.9.6",
+    "@types/react-router": "^5.0.1",
+    "@types/react-router-dom": "^5.1.5",
+    "babel-eslint": "^10.0.1",
+    "babel-jest": "^25.3.0",
+    "babel-loader": "^8.0.2",
+    "babel-plugin-import": "^1.13.0",
+    "babel-polyfill": "^6.26.0",
+    "babel-runtime": "^6.26.0",
+    "clean-webpack-plugin": "^0.1.19",
+    "cross-env": "^5.2.0",
+    "css-loader": "^1.0.0",
+    "cssnano": "^4.1.0",
+    "enzyme": "^3.11.0",
+    "enzyme-adapter-react-16": "^1.15.2",
+    "file-loader": "^2.0.0",
+    "fsevents": "^2.1.3",
```

Comment:
This package is really not used. It can be removed.
I remember adding this package earlier to solve some problems.
I forget the specific problem, not the problem in this project.
I forget to clean it up